app/components/EmbedCampaign.tsx

```diff
@@ -11,9 +11,36 @@ import MinimalHeader from "./MinimalHeader";
 import { PAGE_PADDING } from "./PageWidthGrid";
 import { Disclosure } from "@headlessui/react";
 
+const { embedUrl, challengeUrl } = (() => {
+  const iamUrl = process.env.NEXT_PUBLIC_PASSPORT_IAM_URL || "";
+
+  if (iamUrl.includes("review")) {
+    return {
+      embedUrl: "https://embed.review.passport.xyz",
+      challengeUrl: "https://iam.review.passport.xyz/api/v0.0.0/challenge",
```

Comment:
The challenge endpoint is also hosted on iam: https://embed.staging.passport.xyz/embed/challenge